lalrpop/src/lr1/build/test.rs

```diff
@@ -308,3 +308,26 @@ ty: () = {
     let _lr1_tls = Lr1Tls::install(grammar.terminals.clone());
     build_lr1_states(&grammar, nt("ForeignItem")).unwrap();
 }
+
+// Not sure if this is the right spot
```

Comment:
I'd probably remove this in favor of the end-to-end tests. I tend to add things into the LR1 module if it is testing some quirky bit of the LR(1) algorithm in particular, but this is more testing that the tokenizer integrates properly with the LR(1) stuff.